cli/command/utils.go

```diff
@@ -76,6 +77,52 @@ func PrettyPrint(i any) string {
 
 var ErrPromptTerminated = errdefs.Cancelled(errors.New("prompt terminated"))
 
+// DisableInputEcho disables input echo on the provided io.ReadCloser.
+// This is useful when the user provides sensitive information like passwords.
+// The function returns a restore function that should be called to restore the
+// terminal state.
+func DisableInputEcho(in io.ReadCloser) (restore func() error, err error) {
+	ins := streams.NewIn(in)
```

Comment:
```suggestion
func DisableInputEcho(ins *streams.In) (restore func() error, err error) {
```

The only call to this function passes a return value of `cli.In()` as the parameter which is already a `streams.In` so there's no need for the function to double-wrap the reader.